R/Descriptive_Statistics.Rmd

```diff
@@ -0,0 +1,72 @@
+---
+title: "Descriptive_Statistics"
+author: "OMR Group 2"
+date: "30-11-2020"
+mainfont: Garamond
+fontsize: 12pt
+urlcolor: blue
+output: 
+  pdf_document:
+    latex_engine: xelatex
+---
+
+```{r load_packages}
+
+library(ISLR)
+library(tidyverse)
+library(haven)
+library(readxl)
+library(tinytex)
+library(ggplot2)
+library(scales)
+library(wesanderson)
+
+```
+Import data and check data format.
+```{r 1}
+
+data <- read.csv("30-11-2020.csv", header = TRUE)
```

Comment:
Are you sure that this link isn't broken in this way?

Shouldn't `30-11-2020.csv` be `../data/30-11-2020.csv`?

Reply:
Yes okay, last minute change of putting into the 'data' folder does that. I will push again, one sec

Reply:
Aaah! I see! It is gets the data from a zip-file.

I'll write a quick setup function to make it work tomorrow!

Reply:
^^ Never mind, let's see if this wil work!

Reply:
Fixed this!